kalitectl.py

```diff
@@ -571,7 +570,28 @@ def start(debug=False, watch=False, daemonize=True, args=[], skip_job_scheduler=
             'server.socket_port': port
         })
 
+    # idealy we'd have a setting element, but these are imported much later
+    # and it will require a much deeper change
+    idle_timeout = int(os.getenv("KALITE_IDLE_TIMEOUT", 0))
+    if idle_timeout:
+        import threading
+        cherrypy.engine.last_busy = time.time()
+        def check_connections():
+            for thread in threading.enumerate():
+                if thread.getName().startswith("CP Server Thread"):
```

Comment:
Is there any other thread type that sets `worker_busy`? I wonder if that's enough to distinguish the threads